### PR TITLE
github_action

### PR DESCRIPTION
**Pull request by digitronik**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -12,4 +12,4 @@ jobs:
     - name: check virtualization
       run: |
         sudo apt-get install cpu-checker
-        kvm-ok
+        sudo kvm-ok
```
Comment (sbulage) [inline .github/workflows/main.yml:15]: @digitronik I think `sudo` is not required.

Comment (sbulage) [reply .github/workflows/main.yml:15]: ignore this

**Comment by sbulage** [comment]
@digitronik what I find is the Ubuntu machine didn't have capable processor to run kvm activities.

**Comment by digitronik** [comment]
@sbulage yup.. I was enthusiastic about nested virtualization. I'm closing this but you we can move this project on github action

**Pull request closed without merging**